gensrc/thrift/InternalService.thrift

```diff
@@ -308,6 +308,18 @@ struct TExecPlanFragmentResult {
   1: optional Status.TStatus status
 }
 
+struct TExecBatchPlanFragmentsParams {
+  // required in V1
+  1: optional TExecPlanFragmentParams common_param
+  // required in V1
+  2: optional list<TExecPlanFragmentParams> unique_param_per_instance
+}
+
+struct TExecMultiPlanFragmentsResult {
```

Comment:
Same with TExecPlanFragmentResult.

Reply:
Make service `exec_batch_plan_fragment` returns `TExecPlanFragmentResult`?

Reply:
Seems we needn't `TExecMultiPlanFragmentsResult`, it used to declare a thrift RPC service API, which we needn't.